urdfModifiers/core/modification.py

```diff
@@ -83,6 +89,9 @@ def add_position(self, value, absolute):
         """Adds a modification of the position of the origin"""
         self.position = ModificationType(value, absolute)
 
+    def add_joint_type(self, value):
+        self.joint_type = value
```

Comment:
This allows you to type any string into joint type, even invalid types

Maybe we can have an enum for this, like we do for the Axis. What do you think? @CarlottaSartore

Reply:
Yes I agree, I have added the modification in 9a0a6014c3e4ea57092bddf3dc7bfb1e53f4909d and also updated the readem to make it clearer. 
Anyhow the implementation I did still allow the user to specify any kind of string (even though if it is not a joint type urdfpy will complain) since I did not see a way to not do it without breaking the coherence with the other methods that do not specify the input types. 

**side note** I do not know why even though  the branch in my fork add_joint_type has 4 commits, in this PR I am not able to see commit 9a0a6014c3e4ea57092bddf3dc7bfb1e53f4909d